yb-voyager/src/query/queryparser/traversal_proto.go

```diff
@@ -53,7 +53,10 @@ const (
 	PG_QUERY_JSON_IS_PREDICATE_NODE          = "pg_query.JsonIsPredicate"
 	PG_QUERY_VIEWSTMT_NODE                   = "pg_query.ViewStmt"
 	PG_QUERY_COPYSTSMT_NODE                  = "pg_query.CopyStmt"
-	PG_QUERY_CONSTRAINT_NODE                 = "pg_query.Constraint"
+
+	PG_QUERY_DEFINE_STMT_NODE = "pg_query.DefineStmt"
+	PG_QUERY_MERGE_STMT_NODE  = "pg_query.MergeStmt"
```

Comment:
nit: `PG_QUERY_DEFINESTMT_NODE` and `PG_QUERY_MERGESTMT_NODE` ?

Reply:
similarly for PG_QUERY_JSON_IS_PREDICATE_NODE, lets just keep it consistent, either choice is fine.